src/Image.php

```diff
@@ -1509,6 +1509,7 @@ public function bandand(): Image
      */
     public function bandor(): Image
     {
+        // phpdoc hates OperationBoolean::OR, so use the string form here
```

Comment:
I don't think so, phpcs says that `OperationBoolean::OR` is a syntax error and stops parsing.